src/main/java/com/team4/testingsystem/services/impl/AnswerServiceImpl.java

```diff
@@ -0,0 +1,60 @@
+package com.team4.testingsystem.services.impl;
+
+import com.team4.testingsystem.dto.AnswerRequest;
+import com.team4.testingsystem.entities.Answer;
+import com.team4.testingsystem.exceptions.AnswerNotFoundException;
+import com.team4.testingsystem.exceptions.QuestionNotFoundException;
+import com.team4.testingsystem.repositories.AnswerRepository;
+import com.team4.testingsystem.repositories.QuestionRepository;
+import com.team4.testingsystem.services.AnswerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AnswerServiceImpl implements AnswerService {
+
+    AnswerRepository answerRepository;
+    QuestionRepository questionRepository;
```

Comment:
Let's use `QuestionService` instead of repository. Then you won't need to take care of checking if question exists and throwing exception.